.travis.yml

```diff
@@ -40,6 +40,10 @@ jobs:
     - $TRAVIS_BUILD_DIR/scripts/travis/build.sh
     - docker run --rm -it -v $TRAVIS_BUILD_DIR:/work -w /work sqlflow:dev
       pre-commit run -a --show-diff-on-failure
+  - env:
+    os: osx
+    script:
+    - $TRAVIS_BUILD_DIR/scripts/travis/deploy_cli_macos.sh
```

Comment:
Maybe move this to the `Deploy` stage?

Reply:
Done.

I put it in the first stage only for debugging. Now it is in the last (deploy) stage.